src/sidebar/store/modules/test/annotations-test.js

```diff
@@ -12,14 +17,181 @@ const { actions, selectors } = annotations;
  * Create a Redux store which handles annotation, selection and draft actions.
  */
 function createStore() {
-  return createStoreFromModules([annotations, selection, drafts, viewer], [{}]);
+  return createStoreFromModules(
+    [annotations, selection, drafts, groups, session, viewer],
```

Comment:
I don't have a plan for dealing with this yet, but the fact that the tests for one of the store modules can end up requiring the majority of the other store modules is a red flag.

Reply:
I agree with extreme wholeheartedness. For the record, the dependencies are primarily on the  _state_, not the other store modules' selectors/actions (as much, anyway).